crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -162,6 +162,8 @@ pub enum ConsensusCertificateResult {
     ),
 }
 
+// Unused type.
+// TODO: remove this type.
```

Comment:
maybe add a note to these TODOs about what is preventing the type from being removed rn so a future maintainer can more easily gauge whether it is safe?

Reply:
Thought about this a bit more. This type can actually be removed now.